zerver/lib/actions.py

```diff
@@ -3669,18 +3669,19 @@ def get_emails_from_user_ids(user_ids):
     # We may eventually use memcached to speed this up, but the DB is fast.
     return UserProfile.emails_from_ids(user_ids)
 
-def do_get_realm_aliases(realm):
+def get_realm_aliases(realm):
     # type: (Realm) -> List[Dict[str, Text]]
-    return list(realm.realmalias_set.values('id', 'domain'))
+    return list(realm.realmalias_set.values('id', 'domain', 'subdomains_allowed'))
```

Comment:
Interesting! Zulip doesn't seem to currently use this in its codebase, but it does seem semantically appropriate. Let's just remove the 'id', then.